qiskit/visualization/timeline/interface.py

```diff
@@ -25,21 +25,26 @@
 from qiskit.exceptions import MissingOptionalLibraryError
 from qiskit.visualization.exceptions import VisualizationError
 from qiskit.visualization.timeline import types, core, stylesheet
+from qiskit.utils import deprecate_arg
 
 
+@deprecate_arg("show_idle", new_alias="idle_wires", since="1.1.0")
+@deprecate_arg("show_barriers", new_alias="plot_barriers", since="1.1.0")
 def draw(
     program: circuit.QuantumCircuit,
     style: Optional[Dict[str, Any]] = None,
     time_range: Tuple[int, int] = None,
     disable_bits: List[types.Bits] = None,
     show_clbits: Optional[bool] = None,
-    show_idle: Optional[bool] = None,
-    show_barriers: Optional[bool] = None,
+    idle_wires: Optional[bool] = None,
+    plot_barriers: Optional[bool] = None,
```

Comment:
Per the deprecation policy we need to add `plot_barriers` and `idle_wires`  before we start emitting a deprecation warning here. This way users have a release of overlap where they can use the old and new without a deprecation warning being emitted. If `deprecate_arg` has a `pending` arg setting that would be enough, otherwise we'll have to remove the decorators and just add the new arguments.

Reply:
I added the `pending=True` in f4f6a644f784e91db94c45afd347d7f74a2c7c63